index.js

```diff
@@ -81,9 +82,24 @@ Watchify.prototype.build = function () {
   var outputDir = path.dirname(this.options.outputFile);
   var outputFile = destDir + '/' + this.options.outputFile;
 
+  var browserifyPaths = [];
+
+  var nodeModulesDir = findUp.sync('node_modules', {
```

Comment:
i think we should just suggest users explicitly configure where there node_modules are. Thoughts?

Reply:
Hmm, my idea here was to mimic node's default require algorithm - it's really such an established standard. But are you thinking that it's risky to rely on `process.cwd()`?

Reply:
FYI @stefanpenner – other plugins, like broccoli-sass, implicitly depend on a specific working directory for correctly looking up include files. It might be good to formalise the fact that some plugins will depend on a specific working directory, but after having thought it through, it doesn't seem like a very strange assumption to me that the working directory should be the same directory as where the `Brocfile` is placed.

Reply:
Yes we should formalize. As implemented this will be brittle if:

* tmp dir moves around
* users us symlinks (npm link) etc

Reply:
Alright! What do you think is the best way forward for this PR? Do we formalise a way for Broccoli plugins to deal with `process.cwd()`, or do we do something else?

How come this will be brittle if users use symlinked npm modules?